src/content/app/tools/vep/views/vep-submissions/listed-vep-submission/ListedVepSubmission.module.css

```diff
@@ -71,3 +70,15 @@
 .statusFailed {
   color: var(--color-red);
 }
+
+.speciesName [data-part="common-name"] {
+  font-size: 14px;
+}
+
+.speciesName [data-part="scientific-name"] {
+  font-size: 14px;
+}
+
+.speciesName [data-part="assembly-name"] {
+  font-size: 11px;
+}
```

Comment:
I don't like the repetition of these styles in two files.

- Should the species name in the VEP submission form (i.e. right after you've selected a species) also have the same styles?
- We are probably looking at a new component here. At least one specific to VEP (and to be placed in the vep/components directory). Perhaps even one that should be common across all tools (both VEP and BLAST); but we can start with VEP.

The repetition of these styles in two files suggests that we may be dealing with a